examples/jsm/renderers/common/Background.js

```diff
@@ -50,11 +50,11 @@ class Background extends DataMap {
 
 			if ( backgroundMesh === undefined ) {
 
-				const backgroundMeshNode = context( vec4( backgroundNode ).mul( backgroundIntensity ), {
+				const backgroundMeshNode = context( vec4( backgroundNode ), {
 					// @TODO: Add Texture2D support using node context
 					getUV: () => normalWorld,
 					getTextureLevel: () => backgroundBlurriness
-				} );
+				} ).mul( backgroundIntensity );
```

Comment:
The problem in this model that returns a `OperatorNode`, become incompatible with change of material after first compilation because `backgroundMeshNode` should be a `ContextNode`.

Could you revert this part and add the `mul()` in: 

```js
sceneData.backgroundMeshNode.node = vec4( backgroundNode ).mul( backgroundIntensity );
```
  https://github.com/mrdoob/three.js/blob/d97cc91c4fb3d1824420eee39bf363163fc13295/examples/jsm/renderers/common/Background.js#L86